solr/core/src/test/org/apache/solr/servlet/PrometheusMetricsServletTest.java

```diff
@@ -18,11 +18,12 @@
 
 import java.io.PrintWriter;
 
+import org.apache.solr.core.CoreContainer;
 import org.junit.Test;
 
 public class PrometheusMetricsServletTest {
   @Test
   public void testWriteMetrics() throws Exception {
-    PrometheusMetricsServlet.writeStats(new PrintWriter(System.out));
+    PrometheusMetricsServlet.writeStats(new PrintWriter(System.out), null);
```

Comment:
No assertions or anything?

Reply:
I didn't write this test, I just added an extra param